web/src/App.tsx

```diff
@@ -5,10 +5,10 @@ import * as Sentry from '@sentry/react';
 import { useGetAppVersion } from 'api/getAppVersion';
 import useGetState from 'api/getState';
 import LoadingOverlay from 'components/LoadingOverlay';
+import MapCardContainer from 'components/MapCardContainer';
```

Comment:
Do you think there is enough code in this bundle to justify making it a dynamic import and use code splitting?

Reply:
I moved the MapCardContainer into a suspense boundary so we at least have the same as before!